<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowAdd</class>
 <widget class="QWidget" name="WindowAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление</string>
  </property>
  <widget class="QPushButton" name="PhotoAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Прикрепить фото</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Название:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введите название уведомления</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>441</width>
     <height>236</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">alternate-background-color: rgb(0, 10, 202);
border-color: rgb(0, 0, 0);
selection-color: rgb(0, 0, 0);
background-color: rgb(232, 232, 116);
color: rgb(0, 0, 0);
selection-background-color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дата окончания:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Period">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>600</y>
     <width>321</width>
     <height>61</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Выберите периодичность уведомлений</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Слабая</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Средняя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Сильная</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="Comment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>441</width>
     <height>51</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Введиие комментарий к уведомлению</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelComment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Комментарий</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnAdd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>680</y>
     <width>441</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 0);</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
